### DISCUSSION TITLE
Running Qusetion

### DISCUSSION THREAD

**Discussion by LinKaiyi-design** [firemodels/fds#12583] [Q&A]
I am running FDS into the following problem, can someone help me see how this problem should be solved

I executed the following command on my laptop：
fds_local -p 1 -o 12 L2F.fds
my FDS file is:
L2F.fds
&HEAD CHID='L2F'/
&TIME T_END=3600/
&DUMP DT_RESTART=300.0, DT_DEVC=0.02,DT_SL3D=10,NFRAMES=18000/
&MISC TMPA= 24,HUMIDITY= 60,LEVEL_SET_MODE=4,/
&WIND speed=3,DIRECTION=130 /
&REAC FUEL='CELLULOSE', C=6, H=10, O=5, SOOT_YIELD=0.01, HEAT_OF_COMBUSTION=16400. /
&MESH ID='Mesh01', IJK=353,358,70, XB=0.0,1765.0,0,1790.0,0.0,350.0/ 
&VENT ID='Mesh Vent: Mesh01 [XMAX]', SURF_ID='OPEN', XB=1765.0,1765.0,0,1690.0,0.0,350.0, COLOR='INVISIBLE'/
&VENT ID='Mesh Vent: Mesh01 [XMIN]', SURF_ID='OPEN', XB=0.0,0.0,0,1790.0,0.0,350.0, COLOR='INVISIBLE'/
&VENT ID='Mesh Vent: Mesh01 [YMAX]', SURF_ID='OPEN', XB=0.0,1765.0,1790.0,1790.0,0.0,350.0, COLOR='INVISIBLE'/
&VENT ID='Mesh Vent: Mesh01 [YMIN]', SURF_ID='OPEN', XB=0.0,1765.0,0.0,0.0,0.0,350.0, COLOR='INVISIBLE'/
&VENT ID='Mesh Vent: Mesh01 [ZMAX]', SURF_ID='OPEN', XB=0.0,1765.0,0,1790.0,350.0,350.0, COLOR='INVISIBLE'/
....

my running error:
 Time Step:   1600, Simulation Time:    298.90 s
[mpiexec@LEGIONY9000P] HYD dmx wait for_event (..(windows\srchydra_demux.c:231): unable to post next accept
[mpiexec@LEGIONY9000P] wmain(mpiexec.c:2086):error waiting for event

**Comment by Er9y714** [comment]
The mesh has 8+ million cells. The PC you are running this calculation might have run out of memory. Monitor the RAM usage. 

Secondly the code seems to be generated for older version of FDS. Try running it on the latest version.

**Reply by LinKaiyi-design** [reply]
Thank you very much! I will try to run this file on HPC and use the latest version.
